src/binding/mpi_standard_api.txt

```diff
@@ -1621,23 +1621,23 @@ MPI_Pack_size:
     comm: COMMUNICATOR, [communicator argument to packing call]
     size: POLYNUM_BYTES_NNI, direction=out, [upper bound on size of packed message, in bytes]
 MPI_Parrived:
-    request: REQUEST, direction=INOUT, [partitioned communication request]
+    request: REQUEST, direction=IN, [partitioned communication request]
     partition: PARTITION, direction=IN, [partition to be tested]
     flag: LOGICAL_BOOLEAN, direction=out, [true if operation completed on the specified partition, false if not]
 MPI_Pcontrol:
     level: PROFILE_LEVEL, constant=True, [Profiling level]
     varargs: VARARGS
 MPI_Pready:
     partition: PARTITION, direction=IN, [partition to mark ready for transfer]
-    request: REQUEST, direction=INOUT, [partitioned communication request]
+    request: REQUEST, direction=IN, [partitioned communication request]
 MPI_Pready_list:
     length: ARRAY_LENGTH, direction=IN, [list length]
-    array_of_partitions: PARTITION, direction=INOUT, length=length, [array of partitions]
-    request: REQUEST, direction=INOUT, [partitioned communication request]
+    array_of_partitions: PARTITION, direction=IN, length=length, [array of partitions]
```

Comment:
I should split this line out, since it was always wrong whereas the request parameters were changed recently.